web/src/theme/_jobs.scss

```diff
@@ -1,3 +1,12 @@
 .multiline {
-  white-space: pre;
+  overflow-wrap: break-word;
+  white-space: pre-line;
+  width: 500px;
```

Comment:
Would max-width do the trick ?
Another unit than pixel ? 

Any hint on how to implement this is welcome :)